src/components/User/user.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+
+class User extends Component {
+  constructor(){
+    super();
+    this.state = {
+      user:{}
+    }
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:5000/api/users/moneyJump')
+      .then(res => res.json(console.log(res)))
+      .then(user => this.setState({user: user}, () => console.log('Users fetched...', user)))
+  }
+
+  render() {
+    return (
+      <div>
+        <h2>MoneyJump</h2>
+        <ul>
+          <li key={this.state.user.id}>
+            <p>{this.state.user.name}</p>
+            <img src={this.state.user.image} alt="Logo"/>
```

Comment:
We could the JS concept of Object Destructuring to have a cleaner code 

Reference: [ES6 Destructuring: The Complete Guide](https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f)

Reply:
ok, thanks for sharing the resource, I will read it and try to apply it